core/src/main/java/org/infinispan/manager/EmbeddedCacheManager.java

```diff
@@ -278,7 +291,9 @@
     * memory and in any backing cache store.
     *
     * @param cacheName name of cache to remove
+    * @deprecated obtain a {@link org.infinispan.commons.api.CacheContainerAdmin} instance using {@link #administration()} and invoke the {@link org.infinispan.commons.api.CacheContainerAdmin#removeCache(String)} method
     */
+   @Deprecated // since 9.2
    void removeCache(String cacheName);
```

Comment:
there are sill usage of this method in the server. I guess they can be removed/upgraded.

Reply:
I have removed the usage. Still there for abstract wrappers though.